src/inspect_ai/util/_sandbox/limits.py

```diff
@@ -29,7 +29,7 @@ def verify_exec_result_size(exec_result: ExecResult[str]) -> None:
     """Verify the size of the output streams in an `ExecResult`.
 
     Raises:
-      OutputLimitExceededError: If an output stream exceeds the 1 MiB limit.
+      OutputLimitExceededError: If an output stream exceeds the limit.
```

Comment:
I thought this was "internal" enough to avoid repeating the actual limit value here.